dotCMS/src/main/java/com/dotcms/contenttype/model/field/FieldTypeResource.java

```diff
@@ -43,8 +47,12 @@ public FieldTypeResource(final WebResource webresource, FieldTypeAPI fieldTypeAP
     public Response getFieldTypes(@Context final HttpServletRequest req) {
 
         final InitDataObject initData = this.webResource.init(null, true, req, true, null);
-        User user = initData.getUser();
+        final User user = initData.getUser();
+
+        final ImmutableList<Map<String, Object>> fieldTypesMap = fieldTypeAPI.getFieldTypes(user).stream()
+                .map(fieldType -> fieldType.toMap())
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Specify a type for: 'fieldType' [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2211)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)

Reply:
filedType::toMap